pkg/util/mount/mount_unsupported.go

```diff
@@ -18,20 +18,20 @@ limitations under the License.
 
 package mount
 
-type Mounter struct{}
+type LinuxMounter struct{}
```

Comment:
and this is why it was not called "LinuxMounter" before.  The abstraction is that every OS provides a Mounter type.  It's a 'weak symbol' of sorts. or a compile-time interface.